galerautils/src/gu_enc_utils.hpp

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2022 Percona LLC and/or its affiliates. All rights
+   reserved.
+
+   This program is free software; you can redistribute it and/or
+   modify it under the terms of the GNU General Public License
+   as published by the Free Software Foundation; version 2 of
+   the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA */
+
+#ifndef __GU_ENCRYPTION__
+#define __GU_ENCRYPTION__
+
+#include <string>
+#include <functional>
+
+namespace gu {
+class UUID;
+class Config;
+
+#define ptr2ull(ptr) ((unsigned long long)ptr)
+
+std::string encode64(const std::string& binary);
+std::string decode64(const std::string& base64);
```

Comment:
Also just a stylistic comment, but this pull request uses multiple styles for functions/classes, sometimes even within a single file.

Like here, we have two function stating with small letters, and then 4 below using CamelCase.

Reply:
All method names were changed to `underscore_convention()`.
private and public methods follow the same pattern.
Only constructor/destructor use CammelCase

It looks like this is the Galera convention, so let's stick to it, indeed.